tutorials/install-distro.md

```diff
@@ -0,0 +1,75 @@
+# Install with Package Managers
+
+CRI-O builds for native package managers using [openSUSE's OBS](build.opensuse.org)
```

Comment:
```suggestion
CRI-O builds for native package managers using the [openSUSE's OBS](build.opensuse.org)
```

Reply:
I'm using possessive here, i don't think the is correct